src/utils/file_upload.rs

```diff
@@ -530,27 +534,73 @@ fn print_upload_context_details(context: &UploadContext) {
     );
 }
 
-#[test]
-fn test_url_to_bundle_path() {
-    assert_eq!(url_to_bundle_path("~/bar").unwrap(), "_/_/bar");
-    assert_eq!(url_to_bundle_path("~/foo/bar").unwrap(), "_/_/foo/bar");
-    assert_eq!(
-        url_to_bundle_path("~/dist/js/bundle.js.map").unwrap(),
-        "_/_/dist/js/bundle.js.map"
-    );
-    assert_eq!(
-        url_to_bundle_path("~/babel.config.js").unwrap(),
-        "_/_/babel.config.js"
-    );
+#[cfg(test)]
+mod tests {
+    use sha1_smol::Sha1;
 
-    assert_eq!(url_to_bundle_path("~/#/bar").unwrap(), "_/_/#/bar");
-    assert_eq!(url_to_bundle_path("~/foo/#/bar").unwrap(), "_/_/foo/#/bar");
-    assert_eq!(
-        url_to_bundle_path("~/dist/#js/bundle.js.map").unwrap(),
-        "_/_/dist/#js/bundle.js.map"
-    );
-    assert_eq!(
-        url_to_bundle_path("~/#foo/babel.config.js").unwrap(),
-        "_/_/#foo/babel.config.js"
-    );
```

Comment:
What is it with these nonsensical diffs

Reply:
<img width="230" alt="image" src="https://github.com/getsentry/sentry-cli/assets/1523305/6c5731f7-9c89-4215-b885-c7ef9bbd261c">
and they will be gone :)

Reply:
Just switching on "Hide whitespace" helps a lot in this case, but the question remains why this diff would be generated, with some insertions in the middle of a huge block of deletions.

Reply:
Ah, because you nested it all in `mod tests` now :)